lib/rubocop/cop/rspec/shared_examples.rb

```diff
@@ -53,7 +53,7 @@ class SharedExamples < Base
 
         def on_send(node)
           shared_examples(node) do
-            ast_node = node.first_argument
+            next unless (ast_node = node.first_argument)
```

Comment:
question? Would we instead add argument capturing in the node_matches and not match on shared groups / includes with no argument?

Reply:
Good idea. Sorry I merged too fast 😅